1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -1,89 +1,89 @@
 
-This was the case until 2009 when ECMAScript 5 (ES5) appeared. It added new features to the language and modified some of the existing ones. To keep the old code working, most such modifications are off by default. You need to explicitly enable them with a special directive: `"use strict"`.
+Acest aspect a fost valabil până în 2009, când a apărut ECMAScript 5 (ES5). Acesta a adăugat noi caracteristici limbajului și a modificat unele dintre cele existente. Pentru ca vechiul cod să rămână funcțional, majoritatea acestor modificări sunt dezactivate în mod standard. Trebuie să le activați în mod explicit cu o comandă specială: `"use strict"`.
 
 ## "use strict"
 
-The directive looks like a string: `"use strict"` or `'use strict'`. When it is located at the top of a script, the whole script works the "modern" way.
+Comanda arată ca un string ("șir de caractere"): `"use strict"` sau `'use strict'`. Atunci când este amplasată în partea de sus a unui script, întregul script funcționează în mod "modern".
 
-For example:
+Exemplu:
 
 ```js
 "use strict";
 
-// this code works the modern way
+// acest cod funcționează folosind metoda modernă
 ...
 ```
 
-Quite soon we're going to learn functions (a way to group commands), so let's note in advance that `"use strict"` can be put at the beginning of a function. Doing that enables strict mode in that function only. But usually people use it for the whole script.
+Destul de curând vom învăța functions ("funcții") (o modalitate de a grupa comenzi), așa că putem reține în prealabil că `"use strict"`poate fi pus la începutul unei funcții. Făcând acest lucru, se activează modul strict numai în acea funcție. Dar, de obicei, oamenii îl folosesc pentru întregul script.
 
-````warn header="Ensure that \"use strict\" is at the top"
-Please make sure that `"use strict"` is at the top of your scripts, otherwise strict mode may not be enabled.
+````warn header="Asigurați-vă că \"use strict\" este la început"
+Vă rugăm să vă asigurați că `"use strict"` se află în partea de sus a scripturilor dumneavoastră, în caz contrar, este posibil ca modul strict să nu fie activat.
 
-Strict mode isn't enabled here:
+Modul strict nu este activat aici:
 
 ```js no-strict
 alert("some code");
-// "use strict" below is ignored--it must be at the top
+// "use strict" de mai jos este ignorat - trebuie să fie în partea de sus, la începutul scriptului
 
 "use strict";
 
-// strict mode is not activated
+// modul strict nu este activat
 ```
 
-Only comments may appear above `"use strict"`.
+Doar comentariile pot apărea deasupra `"use strict"`.
 ````
 
-```warn header="There's no way to cancel `use strict`"
-There is no directive like `"no use strict"` that reverts the engine to old behavior.
+```warn header="Nu există nicio modalitate de a anula `use strict`"
+Nu există o comandă de tipul `"no use strict"` care să readucă motorul de căutare ("browserul") la vechiul comportament.
```

Comment:
```suggestion
Nu există o directivă ca `"no use strict"` care să readucă motorul la vechiul comportament.
```

vezi despre `javascript engine`. Asta nu se referă direct la browser, deși acesta are un motor.
Sincer, aș moficica articolul `en` cu:
```
There is no directive like `"no use strict"` that reverts the javascript engine to the old behavior.
```

Reply:
Corect, eu am tradus asa, pentru că fiecare browser are motorul lui care citeste JS. 
Ce părere ai de varianta ”There is no directive like `"no use strict"` that reverts the javascript engine used by browsers to the old behavior.” ?

Reply:
Varianta tradusă `motorul de căutare` se înțelege motor gen Google, Bing etc.
vezi [Motor de căutare](https://ro.wikipedia.org/wiki/Motor_de_c%C4%83utare)

> Ce părere ai de varianta ”There is no directive like "no use strict" that reverts the javascript engine used by browsers to the old behavior.” ?

E ok. Poți modifica și trimite un PR la repozitoriul original din limba engleză.

Reply:
Am trimis un pull request pe limba engleza.